sql/values.go

```diff
@@ -37,8 +39,18 @@ func (p *planner) Values(n parser.Values) (planNode, error) {
 		if !ok {
 			return nil, fmt.Errorf("expected a tuple, but found %T", data)
 		}
+		if len(v.rows) == 0 {
+			nCols = len(vals)
+		} else if nCols != len(vals) {
+			return nil, fmt.Errorf("VALUES lists must all be the same length")
```

Comment:
Switched to `errors.New` but I'm hesitant to place it in a global var. Doing so seems like premature optimization for a message that is used once. IMO, it is better to keep the error message next to the code where the error is generated.